components/scripts/lib/cli-parsers/common.m4

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env bash
 
-# ARG_OPTIONAL_BOOLEAN([enable-gradle-enterprise],[e],[])
+# ARG_OPTIONAL_SINGLE([gradle-enterprise-server],[s],[],[])
+# ARG_OPTIONAL_SINGLE([git-repo],[r],[])
 # ARG_OPTIONAL_SINGLE([git-branch],[b],[])
 # ARG_OPTIONAL_SINGLE([git-commit-id],[c],[])
 # ARG_OPTIONAL_SINGLE([git-options],[o],[])
-# ARG_OPTIONAL_SINGLE([gradle-enterprise-server],[s],[],[])
-# ARG_OPTIONAL_SINGLE([git-repo],[r],[])
 # ARG_OPTIONAL_SINGLE([project-dir],[p],[],[])
+# ARG_OPTIONAL_SINGLE([args],[a],[])
+# ARG_OPTIONAL_BOOLEAN([enable-gradle-enterprise],[e],[])
 # ARG_OPTIONAL_BOOLEAN([interactive],[i],[],[off])
 # ARG_OPTIONAL_BOOLEAN([debug],[],[],[off])
```

Comment:
@jthurne why does it work that there are cmd line arguments defined in this common file that are not used by all scripts?